editor.planx.uk/src/@planx/components/shared/Radio/DescriptionRadio.tsx

```diff
@@ -12,27 +13,45 @@ export interface Props {
   onChange: RadioProps["onChange"];
 }
 
-const StyledFormLabel = styled(FormLabel)(({ theme }) => ({
-  border: "2px lightgray solid",
+interface StyledFormLabelProps {
+  isSelected: boolean;
+}
+
+const StyledFormLabel = styled(FormLabel, {
+  shouldForwardProp: (prop) => prop !== "isSelected",
+})<StyledFormLabelProps>(({ theme, isSelected }) => ({
+  border: "2px solid",
+  borderColor: isSelected
+    ? theme.palette.primary.main
+    : theme.palette.secondary.main,
   padding: theme.spacing(1.5),
   cursor: "pointer",
   display: "block",
   height: "100%",
+  color: theme.palette.text.primary,
+  "& > p": {
+    color: theme.palette.text.secondary,
+  },
 }));
 
 const DescriptionRadio: React.FC<Props> = ({
   title,
   description,
   onChange,
   id,
-}) => (
-  <StyledFormLabel focused={false}>
-    <Box sx={{ paddingBottom: 2, display: "flex", alignItems: "center" }}>
-      <Radio value={id} onChange={onChange} />
-      <Typography>{title}</Typography>
-    </Box>
-    <Typography variant="body2">{description}</Typography>
-  </StyledFormLabel>
-);
+}) => {
+  const radioGroupState = useRadioGroup();
+  const isSelected = radioGroupState?.value === id;
```

Comment:
@ianjon3s (For future reference) This is how I resolved the issues you were facing with getting the selected radio in order to apply a "selected" style. Docs: https://mui.com/material-ui/react-radio-button/#useradiogroup